### PR TITLE
add logging success colour

### PR DESCRIPTION
**Pull request by bwp91**
I have been thinking about doing this for a while - adding a 'green' logging colour available to plugins to highlight _successful_ actions that might otherwise be missed if white and in the middle of another plugin's spam.

What do you guys think?

### CODE REVIEW

File: src/logger.spec.ts
Comment (bauer-andreas) [inline src/logger.spec.ts:1]: Thank you for adding new unit tests to the project 🥳

File: src/logger.ts [hunk 1]
```diff
@@ -152,6 +161,9 @@ export class Logger {
 
     let loggingFunction = console.log;
     switch (level) {
+      case LogLevel.SUCCESS:
+        message = chalk.green(message);
+        break;
```
Comment (bauer-andreas) [inline src/logger.ts:166]: Great addition thanks 👍

**Review by bauer-andreas — APPROVED** [review]
Sounds like a cool idea, happy to have it merged.

Just one thing, as you are trying to merge this into the 1.7.1 beta branch. I would consider this to be part of a minor release, so should we aim for a 1.8.0 beta here?

**Comment by bwp91** [comment]
I was thinking to merge this into the beta-1.7.1 branch but when we end up releasing the new version it should be bumped to 1.8.0.

**Comment by bauer-andreas** [comment]
> I was thinking to merge this into the beta-1.7.1 branch but when we end up releasing the new version it should be bumped to 1.8.0.

Sounds good 👍

**Pull request merged**